site/docs/es/plugins/conversations.md

```diff
@@ -676,16 +676,16 @@ if (conversation.random() < 0.5) { /* hace cosas */ }
 
 ### Regla III: Usar funciones de conveniencia
 
-Hay un montón de cosas instaladas en `conversación` que pueden ayudarte mucho.
-Tu código no se rompe exactamente si no las usas, pero puede ser lento o comportarse de forma confusa.
-Sin embargo, el usuario final puede no notar la diferencia.
+Hay un montón de cosas instaladas en `conversation` que pueden ayudarte mucho.
+Tu código a veces ni siquiera se rompe si no las usas, pero incluso así puede ser lento o comportarse de forma confusa.
 
 ```ts
-// Dormir a través de la conversación, mejora masiva del rendimiento
-await conversation.sleep(3000); // 3 segundos
-
-// Depurar el registro a través de la conversación, no imprime registros confusos
-conversation.log("Hola, mundo");
+// `ctx.session` sólo persiste los cambios para el objeto de contexto más reciente
+conversation.session.myProp = 42; // ¡más fiable!
+// Date.now() puede ser impreciso dentro de las conversaciones
+await conversation.now(); // ¡más preciso!
+// Debug logging via conversation, no imprime logs confusos
+conversation.log("Hola, mundo"); // ¡más transparente!
```

Comment:
```suggestion
// `ctx.session` sólo persiste los cambios para el objeto de contexto más reciente
conversation.session.myProp = 42; // ¡más fiable!

// Date.now() puede ser impreciso dentro de las conversaciones
await conversation.now(); // ¡más preciso!

// Debug logging via conversation, no imprime logs confusos
conversation.log("Hola, mundo"); // ¡más transparente!
```

Reply:
This is probably correct, too. The newlines are put that way in the English original. @habemuscode?